.github/workflows/test.yml

```diff
@@ -0,0 +1,25 @@
+name: Build and Test
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v2
+    - name: Restore Coursier cache
+      uses: coursier/cache-action@v3
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
```

Comment:
I wonder if we should make it into a build matrix for JDK 8 and 11, as I've seen some funky stuff of library updates sneaking through that don't work on older JVMs